### PR TITLE
my commit

### PR DESCRIPTION
**Pull request by JasonNuhn**

### CODE REVIEW

File: src/arrays.js [hunk 1]
```diff
@@ -5,32 +5,64 @@
...
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
   // You should also pass the index into `cb` as the second argument
   // based off http://underscorejs.org/#each
-};
-
 const map = (elements, cb) => {
+  const newArray = [];
+  for (let i = 0; i < elements.length; i++) {
+    newArray.push(elements[i]);
```
Comment (sperrye) [inline src/arrays.js:20]: Don't forget to pass the elements to your cb. Line 20.

File: src/callbacks.js [hunk 2]
```diff
@@ -32,27 +36,45 @@ firstItem(foods, (firstItem) => {
...
 getLength(foods, (length) => {
   console.log(`The length of the array is ${length}.`);
 });
 
 // Write a function called last which passes the last item of the array into the callback
 // code here
 
+const last = (element, cb) => {
+  cb(element[foods.length-1]);
+}
```
Comment (sperrye) [inline src/callbacks.js:52]: You'll need to pass in foods, or a place holder for the array like (arr), or foods will be undefined. If element is your placeholder, you'll want the index of element, not the index of foods.

File: src/arrays.js (hunk 1 above)
Comment (JasonNuhn) [reply src/arrays.js:20]: This has been fixed.  Thanks!

File: src/closure.js [hunk 3]
```diff
@@ -7,10 +7,16 @@ const counter = () => {
...
+
 const counterFactory = () => {
   // Return an object that has two methods called `increment` and `decrement`.
   // `increment` should increment a counter variable in closure scope and return it.
   // `decrement` should decrement the counter variable and return it.
+  let value = 0;
+  return {
+    increment() { return value += 1; },
+    decrement() { return value -= 1; },
+  };
```
Comment (sperrye) [inline src/closure.js:19]: Another way to increment/decrement is to use the `++` and `--` operators. Example: `value++`

File: src/objects.js [hunk 4]
```diff
@@ -5,22 +5,36 @@ const keys = (obj) => {
...
+  });
 };
 
 const values = (obj) => {
   // Return all of the values of the object's own properties.
   // Ignore functions
   // http://underscorejs.org/#values
+  Object.keys(values).forEach((key) => {
+    return Object.keys(obj[key]);
+  });
```
Comment (sperrye) [inline src/objects.js:21]: You may want to revise line 19 from `Object.keys(values)` to `Object.keys(obj)` as obj is the object you're passing in and want to iterate over with forEach.

**Review by sperrye — COMMENTED** [review]
Completed

**Pull request closed without merging**